src/content/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.14.0
+TBD
+
+### Features
+- Add structured and staged logs for pipelines. Groups logs by stage in the okteto UI.
+- Support custom envvars for main okteto components
+- Support defining custom annotation, labels and envvars in pipeline installer jobs
+- Display CLI version for pipeline installer jobs in the Okteto UI admin panel
+- Add ability to download the Okteto cluster context from the UI
+
+### Improvements
+- Upgrade Okteto CLI to [2.4.1](https://github.com/okteto/okteto/releases/tag/2.4.1)
+- Add better defaults for timestamps in event logs
+- Remove deprecated Google Sign-in Javascript Platform Library in favor or standard OAuth. More info [here](https://developers.googleblog.com/2021/08/gsi-jsweb-deprecation.html)
+- Remove the helm catalog on self-hosted installations by default. This makes installing CRDs not mandatory during cluster creation.
```

Comment:
Can we add a link to https://www.okteto.com/docs/enterprise/administration/configuration/#applications